lib/internal/bootstrap_node.js

```diff
@@ -101,10 +101,15 @@
         delete process.env.NODE_UNIQUE_ID;
       }
 
+      // As a function to avoid unnecessarily loading internal/module.
+      const loadInternalModule = () => NativeModule.require('internal/module');
```

Comment:
@bnoordhuis Updated again, I left the `internalModule` below as it is and just duplicated the require.